community_server/src/Controller/AppRequestsController.php

```diff
@@ -337,14 +341,29 @@ public function listTransactions($page = 1, $count = 25, $orderDirection = 'ASC'
                 $transactions = array_reverse($transactions);
             }
         }
-        return $this->returnJson([
-                'state' => 'success',
-                'transactions' => $transactions,
-                'transactionExecutingCount' => $session->read('Transactions.executing'),
-                'count' => count($transactions),
-                'gdtSum' => $gdtSum,
-                'timeUsed' => microtime(true) - $startTime
-            ]);
+        
+        $state_balance = $stateBalancesTable->find()->where(['state_user_id' => $user['id']])->first();
+        
+        $body = [
+            'state' => 'success',
+            'transactions' => $transactions,
+            'transactionExecutingCount' => $session->read('Transactions.executing'),
+            'count' => count($transactions),
+            'gdtSum' => $gdtSum,
+            'timeUsed' => microtime(true) - $startTime
+        ];
+        $now = new FrozenTime();
+        $body['decay_date'] = $now;
+        
+        if(!$state_balance) {
+            $body['balance'] = 0;
+            $body['decay'] = 0;
```

Comment:
this is not a float. I assume js will be good with zero, but shouldn't it be

```suggestion
            $body['balance'] = 0.0;
            $body['decay'] = 0.0;
```